website_v2/gcloud-powershell-website-v2/www/static/css/style.css

```diff
@@ -1,27 +1,27 @@
 /* TODO: Bootstrap, hit this with the css hammer to make sure it's standard. */
-.TOC {
+.table-of-contents {
     /*light grey color*/
     background-color: #fafafa;
     float:left;
```

Comment:
nit: try to be consistent with formatting. (please fix throughout)

float: left; instead of float:left;